src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/DispCallableMetaObject.cs

```diff
@@ -67,6 +73,8 @@ public override DynamicMetaObject BindSetIndex(SetIndexBinder binder, DynamicMet
             return base.BindSetIndex(binder, indexes, value);
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
```

Comment:
Since this method is private and only called from other methods that are suppressing IL2026, this feels like you should mark this method as `RequiresUnreferencedCode`.